tests/serializers/test_any.py

```diff
@@ -722,3 +722,385 @@ class MyEnum(Enum):
     assert v.to_json({MyEnum.A: 'x'}) == b'{"1":"x"}'
     assert v.to_python(1) == 1
     assert v.to_json(1) == b'1'
+
+
+@pytest.mark.parametrize(
```

Comment:
Can add more tests, hopefully the abundance of tests here gives confidence this is all working as intended 😅